.github/workflows/go.yml

```
@@ -0,0 +1,27 @@
+name: Go
+on: [push, pull_request]
+#  push:
+#    branches: [ main ]
+#    paths: '**.go'
+#  pull_request:
+#    branches: [ main ]
+#    paths: '**.go'
+jobs:
+  Go:
+    name: lint
```

```suggestion
    name: Go
```
This does both linting and testing.